test/integration/post-comment-events.ts

```diff
@@ -100,6 +100,31 @@ describe(`'post_comment' event emitting`, () => {
       });
     });
 
+    describe('Luna want to receive notifications about commented posts', () => {
+      beforeEach(async () => {
+        await luna.update({ preferences: { notifyOfCommentsOnCommentedPosts: true } });
+      });
+
+      it(`should not create notification when there are no comments of Luna`, async () => {
+        const post = await createPost(mars, 'Hello, world!');
+        await createComment(mars, post, 'Comment from Mars');
+
+        await expectUserEventsToBe(luna, []);
+        await expectUserEventsToBe(mars, []);
+      });
+
+      it(`should create notification about comment of Mars after comment of Luna`, async () => {
+        const post = await createPost(mars, 'Hello, world!');
+        await createComment(luna, post, 'Comment from Luna');
+        await createComment(mars, post, 'Comment from Mars');
+
+        await expectUserEventsToBe(luna, [
+          { event_type: ET.POST_COMMENT, created_by_user_id: mars.intId },
+        ]);
+        await expectUserEventsToBe(mars, []);
+      });
+    });
```

Comment:
@davidmz I think we also need an inverted test: "Luna doesn't want to receive notifications"

Reply:
Yes, you right. Added,